components/Item/Card/Card.vue

```diff
@@ -63,6 +63,7 @@
             v-if="overlay"
             class="card-lower-buttons d-flex justify-center align-center"
           >
+            <mark-played-button :item="item" />
```

Comment:
This needs dark so it's visible in light mode.

Reply:
I'm forcing it to be dark in `MarkPlayedButton.vue` is there any point in adding a prop if it is dark everywhere?

Reply:
Then, in it's current status, it will look bad on itemdetails page.

Reply:
Ah good catch, updated 👍